Marlin/Conditionals_post.h

```diff
@@ -413,9 +413,10 @@
     #endif
   #endif
 
-  #define IS_Z2_OR_PROBE(P) (   (PIN_EXISTS(Z2_MIN)      && (P) == Z2_MIN_PIN)       \
-                             || (PIN_EXISTS(Z2_MAX)      && (P) == Z2_MAX_PIN)       \
-                             || (PIN_EXISTS(Z_MIN_PROBE) && (P) == Z_MIN_PROBE_PIN))
+  // Is an endstop plug used for the Z2 endstop or the bed probe?
```

Comment:
Note that this comment is about the code that is here, not the code that used to be here.